Makefile

```diff
@@ -36,11 +33,14 @@ vet: # Vet the code
 	go vet $(CHECK_FILES)
 
 dev: ## Run the development containers
-	# Start postgres first and apply migrations
+	# Start postgres then gotrue
 	docker-compose -f $(DEV_DOCKER_COMPOSE) up -d postgres
-	docker-compose -f $(DEV_DOCKER_COMPOSE) run gotrue sh -c "make migrate_dev"
-	# Actually start the containers for dev
-	docker-compose -f $(DEV_DOCKER_COMPOSE) up
+	docker-compose -f $(DEV_DOCKER_COMPOSE) up -d gotrue
```

Comment:
Without running migrations, these commands could be combined into `docker-compose -f $(DEV_DOCKER_COMPOSE)  up -d` since the services don't need to start seperatly.

Also, I prefer not having a separate down command if it's not needed. Maybe there should be three commands?
`make dev` -> start containers attached
`make up` -> start containers detached
`make down` -> down the containers